core-bundle/src/DependencyInjection/Compiler/RegisterPagesPass.php

```diff
@@ -122,7 +122,7 @@ private function getControllerName(Reference $reference, Definition $definition,
         if (isset($attributes['method'])) {
             $definition->setPublic(true);
 
-            return $controller.':'.$attributes['method'];
+            return $controller.'::'.$attributes['method'];
```

Comment:
hasn't this already been fixed in https://github.com/contao/contao/pull/8463 ?

Reply:
Branch out of date with the base branch 🙃